kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverNames.kt

```diff
@@ -233,58 +228,52 @@ public val TaskContainer.koverLogName: String
 }
 
 @Deprecated(
-    message = "Function was removed, use KoverNames.koverHtmlReportName(buildVariantName) function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
-    replaceWith = ReplaceWith("KoverNames.koverHtmlReportName(buildVariantName)"),
+    message = "Function koverAndroidHtmlReportName was removed, use KoverNames.koverHtmlReportName(buildVariantName) function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
     level = DeprecationLevel.ERROR
 )
 @Suppress("UNUSED_PARAMETER")
 public fun TaskContainer.koverAndroidHtmlReportName(buildVariantName: String): String {
-    throw KoverDeprecationException("Function koverAndroidHtmlReportName(buildVariantName) was removed, use KoverNames.koverHtmlReportName(buildVariantName) function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+    throw KoverDeprecationException("Function koverAndroidHtmlReportName was removed, use KoverNames.koverHtmlReportName(buildVariantName) function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
 }
 
 @Deprecated(
-    message = "Function was removed, use KoverNames.koverBinaryReportName(buildVariantName) function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
-    replaceWith = ReplaceWith("KoverNames.koverBinaryReportName(buildVariantName)"),
+    message = "Function koverAndroidBinaryReportName was removed, use KoverNames.koverBinaryReportName function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
     level = DeprecationLevel.ERROR
 )
 @Suppress("UNUSED_PARAMETER")
 public fun TaskContainer.koverAndroidBinaryReportName(buildVariantName: String): String {
-    throw KoverDeprecationException("Function koverAndroidBinaryReportName(buildVariantName) was removed, use koverBinaryReportName(buildVariantName) function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
+    throw KoverDeprecationException("Function koverAndroidBinaryReportName was removed, use koverBinaryReportName function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}")
 }
 
 @Deprecated(
-    message = "Function was removed, use KoverNames.koverVerifyName(buildVariantName) function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
-    replaceWith = ReplaceWith("KoverNames.koverVerifyName(buildVariantName)"),
+    message = "Function koverAndroidVerifyName was removed, use KoverNames.koverVerifyName(buildVariantName) function. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
```

Comment:
Same problem here and in all other functions: `koverAndroidVerifyName` is mentioned without param, but `KoverNames.koverVerifyName(buildVariantName)` — with